apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -245,7 +246,6 @@ class InlineAudio extends React.Component {
               this.props.style && this.props.style.button,
               this.state.hover && styles.hover
```

Comment:
can we also move the hover logic and the appropriate styling (ie, `cursor: 'pointer'`, etc)?

Reply:
I'll take a look at this. The hover seemed to be working already so maybe I should just remove the old logic for the play part.

Reply:
I think it's just that both the play and "volume" buttons implement the same hover logic and styling, which is silly but at least kind of makes sense if they're supposed to be independent entities.

Reply:
Ya it's working correctly with the hover currently so I'm going to leave it as is